src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -4,4 +4,4 @@ NuGet.Commands.ResolvedDependencyKey.ResolvedDependencyKey() -> void
 static NuGet.Commands.ResolvedDependencyKey.operator !=(NuGet.Commands.ResolvedDependencyKey left, NuGet.Commands.ResolvedDependencyKey right) -> bool
 static NuGet.Commands.ResolvedDependencyKey.operator ==(NuGet.Commands.ResolvedDependencyKey left, NuGet.Commands.ResolvedDependencyKey right) -> bool
 ~static NuGet.Commands.MSBuildRestoreUtility.GetCentralPackageManagementSettings(NuGet.Commands.IMSBuildItem projectSpecItem, NuGet.ProjectModel.ProjectStyle projectStyle) -> (bool IsEnabled, bool IsVersionOverrideDisabled, bool IsCentralPackageTransitivePinningEnabled, bool isCentralPackageFloatingVersionsEnabled)
```

Comment:
Shouldn't this be in shipped since it's breaking change?

I don't think anyone will get affected, but from the perspective of how do we handle these types of issues, I'm not sure this is the correct result.

Reply:
I'm not sure, according to release-6.9.x only the original implementation is shipped (with the nullable):

```
~static NuGet.Commands.MSBuildRestoreUtility.GetCentralPackageManagementSettings(NuGet.Commands.IMSBuildItem projectSpecItem, NuGet.ProjectModel.ProjectStyle? projectStyle) -> (bool IsEnabled, bool IsVersionOverrideDisabled, bool IsCentralPackageTransitivePinningEnabled, bool isCentralPackageFloatingVersionsEnabled)
```

But you can't have that one in shipped and the new one in unshipped.  I have no preference, what do you think we should do?

Reply:
If we're making a breaking change, we change the shipped.txt file.